madmom/evaluation/key.py

```diff
@@ -1,74 +1,7 @@
 from collections import Counter
 
 from . import EvaluationMixin, evaluation_io
-
-_KEY_TO_SEMITONE = {'c': 0, 'c#': 1, 'db': 1, 'd': 2, 'd#': 3, 'eb': 3, 'e': 4,
-                    'f': 5, 'f#': 6, 'gb': 6, 'g': 7, 'g#': 8, 'ab': 8, 'a': 9,
-                    'a#': 10, 'bb': 10, 'b': 11, 'cb': 11}
-
-
-def key_label_to_class(key_label):
-    """
-    Convert key label to key class number.
-
-    The key label must follow the MIREX syntax defined at
-    http://music-ir.org/mirex/wiki/2017:Audio_Key_Detection:
-    `tonic mode`, where tonic is in {C, C#, Db, ... Cb} and mode in {'major',
-    'maj', 'minor', 'min'}. The label will be converted into a class id based
-    on the root pitch id (c .. 0, c# .. 1, ..., cb ... 11) plus 12 if in minor
-    mode.
-
-    Parameters
-    ----------
-    key_label : str
-        Key label.
-
-    Returns
-    -------
-    key_class : int
-        Key class.
-
-    Examples
-    --------
-    >>> from madmom.evaluation.key import key_label_to_class
-    >>> key_label_to_class('D major')
-    2
-
-    >>> key_label_to_class('D minor')
-    14
-
-    """
-    tonic, mode = key_label.split()
-    key_class = _KEY_TO_SEMITONE[tonic.lower()]
-    if mode in ['minor', 'min']:
-        key_class += 12
-    elif mode in ['major', 'maj']:
-        key_class += 0
-    else:
-        raise ValueError('Unknown mode: {}'.format(mode))
-    return key_class
```

Comment:
Shouldn't the `key_label_to_class` function stay in the evaluation module? It seems to me that the integer representation is some kind of internal mapping used for evaluation, isn't it?

Reply:
Well, yes, kind of. Maybe it's cleaner to have the mapping in the evaluation module, right!